lib/internal/modules/esm/get_format.js

```diff
@@ -224,4 +226,5 @@ module.exports = {
   defaultGetFormatWithoutErrors,
   extensionFormatMap,
   extname,
+  kFormatHasNoSource,
```

Comment:
Did you forget to commit something? I don’t see when you’re ever setting `format` to be this symbol.

We don’t want this symbol leaking into user hooks. There shouldn’t be a user `load` hook that ever sees `format` set to this symbol.